applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -4,3 +4,281 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#if (CONFIG_AUDIO_DEV == HEADSET)
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/audio/audio.h>
+
+#include "le_audio.h"
+#include "macros_common.h"
+#include "ctrl_events.h"
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/audio/audio.h>
+#include <bluetooth/audio/capabilities.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(cis_headset, CONFIG_LOG_BLE_LEVEL);
+
+static le_audio_receive_cb receive_cb;
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+};
+
+#define CHANNEL_COUNT_1 BIT(0)
+static struct bt_codec lc3_codec =
+	BT_CODEC_LC3(BT_CODEC_LC3_FREQ_ANY, BT_CODEC_LC3_DURATION_10, CHANNEL_COUNT_1, 40u, 120u,
+		     1u, (BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA),
+		     BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+
+static struct bt_conn *default_conn;
+static struct bt_audio_stream streams;
+static struct bt_audio_stream *stream_cap_config_cb(struct bt_conn *conn,
+					  struct bt_audio_ep *ep,
+					  enum bt_audio_pac_type type,
+					  struct bt_audio_capability *cap,
+					  struct bt_codec *codec)
+{
+	struct bt_audio_stream *stream = &streams;
+
+	if (!stream->conn) {
+		LOG_DBG("ASE Codec Config stream %p", (void *)stream);
+		return stream;
+	}
+
+	LOG_WRN("No streams available");
+	return NULL;
+}
+
+static int stream_cap_reconfig_cb(struct bt_audio_stream *stream,
+			struct bt_audio_capability *cap,
+			struct bt_codec *codec)
+{
+	LOG_DBG("ASE Codec Reconfig: stream %p cap %p", (void *)stream, (void *)cap);
+
+	/* We only support one QoS at the moment, reject changes */
+	return -ENOEXEC;
+}
+
+static int stream_cap_qos_cb(struct bt_audio_stream *stream, struct bt_codec_qos *qos)
+{
+	LOG_DBG("QoS: stream %p qos %p", (void *)stream, (void *)qos);
+
+	return 0;
+}
+
+static int stream_cap_enable_cb(struct bt_audio_stream *stream,
+		      struct bt_codec_data *meta,
+		      size_t meta_count)
+{
+	int ret;
+	struct event_t event;
+
+	LOG_DBG("Enable: stream %p meta_count %u", (void *)stream, meta_count);
+	event.event_source = EVT_SRC_LE_AUDIO;
+	event.le_audio_activity.le_audio_evt_type = LE_AUDIO_EVT_STREAMING;
+
+	ret = ctrl_events_put(&event);
+	ERR_CHK(ret);
+	return 0;
+}
+
+static int stream_cap_start_cb(struct bt_audio_stream *stream)
+{
+	LOG_DBG("Start: stream %p", (void *)stream);
+	return 0;
```

Comment:
Nitpick: Some cb functions use newline after LOG_DBG(), and some do not. Should stick to doing the same in every cb function